web/frontend/src/components/slam.vue

```diff
@@ -48,6 +51,19 @@ const loaded2d = $computed(() => (pointcloud !== undefined && pose !== undefined
 let slam2dTimeoutId = -1;
 let slam3dTimeoutId = -1;
 
+const moveClicked = $computed(() => {
+  for (const element of props.operations) {
+    if (element.op.method.includes('MoveOnMap')) {
```

Comment:
cycle through current ops and return true iff there is an active MoveOnMap method.

Reply:
What causes there to be an active MoveOnMap method?

Reply:
if a user pressed move on the 2D slam ui and the req's callback has not been reached yet indicated we are either still planning or moving the rover.

Reply:
@DTCurrie / @nfranczak do you know:
When does `props.operations` get updated? For example: does props.operations get updated on browser A's remote-control UI for a given robot if browser B initiates a new operation with that robot? 

If so: how does browser B learn about that operation? Is it polling, does viam-serever push all operations to all browsers connected to that robot, etc?

Reply:
> When does props.operations get updated? 

I did `console.log` on `moveClicked` to see how often it was being recomputed.
I saw that it is recomputed ~4x per second!
My understanding is that since `props.operations` is the same as the information which is displayed in the "Operations & Sessions" window is updated within a specified `HeartbeatWindow`.